hs-bindgen/fixtures/bitfields.rs

```diff
@@ -16,10 +16,7 @@ where
     Storage: AsRef<[u8]> + AsMut<[u8]>,
 {
     #[inline]
-    pub fn get_bit(&self, index: usize) -> bool {
-        debug_assert!(index / 8 < self.storage.as_ref().len());
-        let byte_index = index / 8;
-        let byte = self.storage.as_ref()[byte_index];
+    fn extract_bit(byte: u8, index: usize) -> bool {
```

Comment:
I guess the treatment of bitfields must have changed, even though I don't see anything about it in the release notes (https://github.com/rust-lang/rust-bindgen/releases/tag/v0.71.0).